CMakeLists.txt

```diff
@@ -441,92 +442,75 @@ elseif(MIOPEN_BACKEND STREQUAL "OpenCL")
     set(CPACK_RPM_PACKAGE_CONFLICTS miopen-hip)
 endif()
 
-# Unpack DB files
-function(unpack_db db_bzip2_file)
-    set(KERNELS_DIR "${CMAKE_SOURCE_DIR}/src/kernels")
-    STRING(REPLACE ".bz2" "" db_file "${db_bzip2_file}")
-    find_program(UNZIPPER NAMES lbunzip2 bunzip2)
-    if(EXISTS "${db_file}")
-        message(STATUS "WARNING: ${db_file} already exists and will be overwritten")
```

Comment:
[Q] @JehandadKhan Do we no longer need this warning?

Reply:
The warnings are unnecessary - the unzipping has changed, and now it is CMake responsibility to regenerate (override) a file if a source has changed. The files are unpacked into the build directory instead of a source directory (as it was previously). I explained in the PR description why.

Reply:
@atamazov Should I mark the comment resolved?

Reply:
@JehandadKhan @apwojcik Sure [resolved]